packages/celery-library/src/celery_library/rpc/_async_jobs.py

```diff
@@ -22,21 +17,27 @@
     JobSchedulerError,
 )
 from servicelib.celery.models import TaskState
+from servicelib.celery.task_manager import TaskManager
 from servicelib.logging_utils import log_catch
 from servicelib.rabbitmq import RPCRouter
 
-from ...modules.celery import get_task_manager_from_app
+from ..errors import (
+    TransferrableCeleryError,
+    decode_celery_transferrable_error,
+)
 
 _logger = logging.getLogger(__name__)
 router = RPCRouter()
 
 
 @router.expose(reraise_if_error_type=(JobSchedulerError,))
-async def cancel(app: FastAPI, job_id: AsyncJobId, job_id_data: AsyncJobNameData):
-    assert app  # nosec
+async def cancel(
+    task_manager: TaskManager, job_id: AsyncJobId, job_id_data: AsyncJobNameData
```

Comment:
Q: are these routes also exposed in storage's rpc server?

Reply:
Sure, the storage exposes exactly this implementation.